### DISCUSSION TITLE
Works with Emscripten/WASM?

### DISCUSSION THREAD

**Discussion by tim-taylor** [albin-johansson/Centurion#118] [Q&A] [answered]
This looks like a really nice library - I've just discovered it and am looking forward to trying it out. Do you know if programs using it will compile easily with emscripten to wasm?

**Comment by albin-johansson** [comment, marked as answer]
Hi! Thanks for checking it out. 

I'm not very familiar with neither Emscripten nor Wasm, but it shouldn't be a problem as long as C++17 is supported by the toolchain. Centurion uses no other libraries than SDL and the C++ standard library, so that is pretty much the deciding factor.

However, please let me know if there are any issues and I'll check it out.

**Reply by tim-taylor** [reply]
Yes, I think it should work fine. I'll let you know how I get on.
